java/tools/src/test/java/org/apache/arrow/tools/TestIntegration.java

```diff
@@ -342,8 +339,8 @@ public void testInvalid() throws Exception {
       integration.run(args3);
       fail("should have failed");
     } catch (IllegalArgumentException e) {
-      assertTrue(e.getMessage(), e.getMessage().contains("Different values in column"));
-      assertTrue(e.getMessage(), e.getMessage().contains("999"));
+      assertTrue(e.getMessage().contains("Different values in column"), e.getMessage());
```

Comment:
This sort of thing should probably use `assertThrows`, but we don't necessarily have to fix that right now

Reply:
I fixed it.